.circleci/config.yml

```diff
@@ -1,49 +1,80 @@
 # Reference used: https://circleci.com/docs/2.0/language-android
+# _JAVA_OPTIONS reference: https://discuss.circleci.com/t/circle-ci-v2-and-android-memory-issues/11207/9
+#     - (Least invasive change to resolve out-of-memory error)
```

Comment:
Naturally, I prefer the comment where I had it before. 😄

I do think it's a little easier to know what it refers to when it's right next to the settings itself.

Reply:
I moved it here for the split YML, I'll move it back for the straight shot one :)